### DISCUSSION TITLE
Separate* config options with grouping

### DISCUSSION THREAD

**Discussion by gkranis** [renovatebot/renovate#19061] [Suggest an Idea]
Similarly to https://github.com/renovatebot/renovate/discussions/19042 i would like to ask how do the `separate*` options (like `separateMajorMinor`) work in combination with grouping.  Havent experimented yet (since the answers here seem to be fast and i want to save time) but i suspect they get ignored if the update is part of a group. Can someone verify? (meanwhile ill run some experiments of my own)

**Comment by rarkins** [comment]
The separate* commands will influence whether you get major+minor, or only major, or major+minor+patch, etc. After that, grouping is done normally. I don't recommend that you mix separate* settings within the same groups though.
